docs/fsharp/library-reference/core-library-reference/array/toc.yml

```
@@ -0,0 +1,175 @@
+- name: Array Module
+  href: index.md
+  items:
+    - name: Array.Parallel module
+      href: parallel/array.parallel-module.md
+      items:
+        - name: Array.Parallel.choose<'T,'U> Function
+          href: parallel/parallel.choose['t,'u]-function.md
+        - name: Array.Parallel.collect<'T,'U> Function
+          href: parallel/parallel.collect['t,'u]-function.md
+        - name: Array.Parallel.init<'T> Function
+          href: parallel/parallel.init['t]-function.md
+        - name: Array.Parallel.iter<'T> Function
+          href: parallel/parallel.iter['t]-function.md
+        - name: Array.Parallel.iteri<'T> Function
+          href: parallel/parallel.iteri['t]-function.md
+        - name: Array.Parallel.map<'T,'U> Function
+          href: parallel/parallel.map['t,'u]-function.md
+        - name: Array.Parallel.mapi<'T,'U> Function
+          href: parallel/parallel.mapi['t,'u]-function.md
+        - name: Array.Parallel.parition<'T> Function
+          href: parallel/parallel.partition['t]-function.md
+    - name: Array.append<'T> Function
+      href: array.append['t]-function.md
```

Try this:

```suggestion
      href: array.append%5B't%5D-function.md
```

This is how GitHub renders that file name